File: eBotanika.Model/Korisnik.cs
using System;

namespace eBotanika.Model
{
    public class Korisnik
    {
        public int KorisnikID { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public DateTime DatumRodjenja { get; set; }
        public string Email { get; set; }
        public string Telefon { get; set; }
        public string KorisnickoIme { get; set; }
    }
}

File: ebotanika_mobile/lib/models/korisnik.dart
class Korisnik {
  String ime;
  String prezime;
  DateTime datumRodjenja;
  String email;
  String telefon;
  String korisnickoIme;

  Korisnik(
      {required this.ime,
      required this.prezime,
      required this.datumRodjenja,
      required this.email,
      required this.telefon,
      required this.korisnickoIme});

  factory Korisnik.fromJson(Map<String, dynamic> json) {
    return Korisnik(
        ime: json["ime"],
        prezime: json["prezime"],
        datumRodjenja: DateTime.parse(json["datumRodjenja"].toString()),
        email: json["email"],
        telefon: json["telefon"],
        korisnickoIme: json["korisnickoIme"]);
  }

  Map<String, dynamic> toJson() => {
        "ime": ime,
        "prezime": prezime,
        "datumRodjenja": datumRodjenja.toIso8601String(),
        "email": email,
        "telefon": telefon,
        "korisnickoIme": korisnickoIme
      };
}
